docs/data-cookbooks/about-cookbooks.md

```diff
@@ -2,11 +2,11 @@
 
 ### What is a data cookbook?
 
-A data cookbook is a document that helps unify an institution/user group on a set of terms. Additionally, a data cookbook can be a hub of examples and explanations for workflows.
+The following data cookbooks are documents containing examples and explanations of workflows in the Cumulus framework. Additionally, the following data cookbooks should serve to help unify an institution/user group on a set of terms.
 
 ### Adding a page
 
-As shown in detail on in the relevant documentation(1), you can add a new page to the data cookbook by creating a markdown (`.md`) file in the `docs/` directory. The new page can then be linked to the webpage itself by adding it to the `SUMMARY.md` structure.
+As shown in detail on in the relevant [documentation][^1], you can add a new page to the data cookbook by creating a markdown (`.md`) file in the `docs/` directory. The new page can then be linked to the webpage itself by adding it to the glossaries shown in `SUMMARY.md` and `README.md`.
```

Comment:
I don't think [documentation][^1] is doing what you hope it is. 
![screen shot 2018-07-05 at 10 15 00 am](https://user-images.githubusercontent.com/479480/42335045-4bd24000-803c-11e8-9b18-3c1702f0f2dd.png)

I'd either get rid of the brackets around documenation or just make it a direct link instead of a footnote.
` the relevant [documentation](https://toolchain.gitbook.com/pages.html), you can`